Slider/Assets/Scripts/Sliders/Grid/SGrid.cs

```diff
@@ -642,4 +664,9 @@ protected static string GetTileIdAt(int x, int y)
     {
         return Current.targetGrid[(Current.Height - y - 1) * Current.Width + x].ToString();
     }
+
+    public static STile GetTileAt(int x, int y)
```

Comment:
can you move this and `GetTileIdAt()` to be with the other `GetStile(int islandId)` around line 253